models/core/txs.sql

```diff
@@ -25,7 +25,7 @@ FINAL AS (
         tx :from :: STRING AS from_address,
         tx :to :: STRING AS to_address,
         tx :value AS VALUE,
-        tx :block_number AS block_number,
+        tx :block_number AS block_id,
```

Comment:
this will impact any downstream table that have relied on this, have you checked on that?